src/mango/src/mango_execution_stats.hrl

```diff
@@ -0,0 +1,17 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+% http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-record(execution_stats, {
```

Comment:
@tonysun83 ideally the key lookup counter would be as you describe - it's just not implemented that way yet (code assumes key lookups == doc lookups until we have covering indexes). Note that the current counters don't say anything about the number of rows returned - just the number of internal row / document fetches.